go-controller/pkg/ovn/pods.go

```diff
@@ -26,21 +27,42 @@ func (oc *DefaultNetworkController) syncPods(pods []interface{}) error {
 	//
 	// TBD: Before this succeeds, add Pod handler should not continue to allocate IPs for the new Pods.
 	expectedLogicalPorts := make(map[string]bool)
+	vms := make(map[ktypes.NamespacedName]bool)
 	for _, podInterface := range pods {
 		pod, ok := podInterface.(*kapi.Pod)
 		if !ok {
 			return fmt.Errorf("spurious object in syncPods: %v", podInterface)
 		}
 
-		if !oc.isPodScheduledinLocalZone(pod) {
+		switchName, err := oc.getExpectedSwitchName(pod)
+		if err != nil {
+			return fmt.Errorf("failed geting expected switch name at syncPods: %v", err)
+		}
+		if kubevirt.IsPodLiveMigratable(pod) {
+			vm := kubevirt.ExtractVMNameFromPod(pod)
```

Comment:
Could we have multiple pods for the same VM here? should we just care about one of them?

Reply:
yeah but for pods with the same VM, some might be remote and others might be local?

Reply:
We are missing skipping stale pods, if we don't do so this will be wrong
```golang
vms[*vm] = oc.isPodScheduledinLocalZone(pod)
```
Will add that.